app/controllers/session_controller.rb

```diff
@@ -7,10 +10,13 @@ def new
 
   #POST /session
   def create
-    user = User.find_by(email: params[:email])
-    if user && user.authenticate(params[:password])
-      session[:current_user_id] = user.id
-      redirect_to user_url
+    @user = User.find_by(email: params[:email])
+    if @user && @user.authenticate(params[:password])
```

Comment:
this is gr888888, really simple, nice one Jay and Jeff!

Reply:
👍